.github/workflows/load-tests.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           path: ./bin/*
 
   loadtest:
-    runs-on: ubuntu-24.04
+    runs-on: "Linux-x64-16-64GB-Runner"
```

Comment:
If we know the image version (ubuntu-latest or ubuntu-24.04?) we could document it in a comment.

Reply:
done